model/src/main/kotlin/utils/DependencyGraphBuilder.kt

```diff
@@ -219,10 +218,11 @@ class DependencyGraphBuilder<D>(
      * Return a set of all the scope names known to this builder that start with the given [prefix]. If [unqualify] is
      * *true*, remove this prefix from the returned scope names.
      */
-    fun scopesFor(prefix: String, unqualify: Boolean = true): SortedSet<String> {
-        val qualifiedScopes = scopeMapping.keys.filter { it.startsWith(prefix) }
-        val scopes = qualifiedScopes.takeUnless { unqualify } ?: qualifiedScopes.map { it.substring(prefix.length) }
-        return scopes.toSortedSet()
+    fun scopesFor(prefix: String, unqualify: Boolean = true): Set<String> {
```

Comment:
Commit message: Typo in "prece~e~ding" (the stroked through "e" does not really render well)